### DISCUSSION TITLE
Security Onion 2 on AWS

### DISCUSSION THREAD

**Discussion by sebaslml** [Security-Onion-Solutions/securityonion#5984] [Unsupported Versions] [answered]
Hi, i just started my proyect installing an Standalone version on AWS. 
But when I run the instace it just detect 72 GB of storage, whe I added 200 GB of storage which is the minimun to run an standalone version.

If you need any pic I can provide it.

**Comment by dougburks** [comment, marked as answer]
It seems like maybe your 200GB of storage was partitioned to a 72GB volume.

Please note that 200GB is the MINIMUM, meaning that you likely will need more than that anyway.

**Reply by sebaslml** [reply]
I need more help my friend

**Comment by sebaslml** [comment]
But how was partitioned ??? 
this is the error:
![error](https://user-images.githubusercontent.com/92940863/138728772-d3a34ca9-60ad-4202-9723-31bf71113d29.PNG)

I just added 100 GB on storage
![info](https://user-images.githubusercontent.com/92940863/138728790-e3b01b20-cc33-48ae-8aee-1199443fb160.PNG)

Should I type a command on CLI or could be some configuration con aws?

**Reply by sebaslml** [reply]
I already run this command but return the same error

sudo so-prepare-fs

**Reply by jertel** [reply]
If you're using this environment as a temporary/proof of concept then you can proceed past that warning. Since you've used `so-prepare-fs` it will store Elasticsearch data on the second 100GB drive and the rest of the data will be stored on the primary root disk.

If you're planning to use this for a production system with some decent network throughput then I suggest increasing the storage capacity accordingly.

**Reply by sebaslml** [reply]
Great thanks a lot!!!!
It works!
